examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
It's been years since I've done a :validate lambda. Looking at [the current code](https://github.com/elastic/logstash/blob/master/lib/logstash/config/mixin.rb#L340-L482), I'm not even sure lambdas are supported anymore, haha :(

Alternate validation is possible in the plugin register (or initialize) methods with custom code.  One example of this is in the [elasticsearch output](https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/lib/logstash/outputs/elasticsearch.rb#L323-L345)